github/package.json

```diff
@@ -0,0 +1,53 @@
+{
+    "name": "@microsoft/vscode-azext-github",
+    "author": "Microsoft Corporation",
+    "version": "1.0.0",
+    "description": "Common GitHub tools for VS Code",
+    "tags": [
+        "azure",
+        "vscode"
+    ],
+    "keywords": [
+        "azure",
+        "vscode"
+    ],
+    "main": "out/src/index.js",
+    "types": "out/src/index.d.ts",
+    "license": "MIT",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Microsoft/vscode-azuretools"
+    },
+    "bugs": {
+        "url": "https://github.com/Microsoft/vscode-azuretools/issues"
+    },
+    "homepage": "https://github.com/Microsoft/vscode-azuretools/blob/main/github/README.md",
+    "scripts": {
+        "build": "tsc -p ./",
+        "compile": "tsc -watch -p ./",
+        "lint": "eslint --ext .ts .",
+        "lint-fix": "eslint --ext .ts . --fix",
+        "package": "npm pack",
+        "test": "node ./out/test/runTest.js"
+    },
+    "devDependencies": {
+        "@microsoft/eslint-config-azuretools": "^0.2.1",
+        "@types/glob": "^8.1.0",
+        "@types/mocha": "^7.0.2",
+        "@types/node": "^16.0.0",
+        "@types/vscode": "1.76.0",
+        "@typescript-eslint/eslint-plugin": "^5.53.0",
+        "@vscode/test-electron": "^2.1.5",
+        "eslint": "^8.34.0",
+        "eslint-plugin-import": "^2.22.1",
+        "glob": "^7.1.6",
+        "mocha": "^9.1.3",
+        "mocha-junit-reporter": "^2.0.2",
+        "mocha-multi-reporters": "^1.1.7",
+        "typescript": "^5.1.3"
+    },
+    "dependencies": {
+        "@microsoft/vscode-azext-utils": "^1.2.0",
```

Comment:
Lets update to ^2.0.0 if possible

Reply:
Sure, I can bump the version before we merge.  Container Apps is still on 1.2.0 so using this on 2.0.0 was causing some issues and I was trying to test it quickly.